src/pineko/fonll.py

```diff
@@ -0,0 +1,274 @@
+"""Module to manage FONLL predictions."""
+
+import copy
+import json
+import tempfile
+from pathlib import Path
+
+import numpy as np
+import pineappl
+import rich
+import yaml
+
+from . import configs
+
+
+class FONLLInfo:
+    """Class containing all the information for FONLL predictions."""
+
+    def __init__(
+        self, ffns3, ffn03, ffns4, ffns4til, ffns4bar, ffn04, ffns5, ffns5til, ffns5bar
+    ) -> None:
+        """Initialize fonll info."""
+        self.ffns3 = ffns3
+        self.ffn03 = ffn03
+        self.ffns4 = ffns4
+        self.ffns4til = ffns4til
+        self.ffns4bar = ffns4bar
+        self.ffn04 = ffn04
+        self.ffns5 = ffns5
+        self.ffns5til = ffns5til
+        self.ffns5bar = ffns5bar
```

Comment:
Even in this case, a list might be a suitable option.

Or even better: a list of tuples, i.e. `[(m, til, bar, m0), ...]` (at least it's easier to process automatically)